src/commands/sidebar_find_symbol.ts

```diff
@@ -0,0 +1,401 @@
+import {
+  Configuration,
+  Disposable,
+  LanguageClient,
+  lspRangeToRange,
+  NotificationRequest,
+  nova,
+  TreeDataProvider,
+  TreeItem,
+  TreeItemCollapsibleState,
+  TreeView,
+  wrapCommand,
+} from "../nova_utils.ts";
+import { lsp } from "../../deps.ts";
+let symbolDataProvider: SymbolDataProvider | null = null;
+
+interface ElementConversionOptions {
+  shouldDisambiguate?: boolean;
```

Comment:
What does it mean to disambiguate in this context? I read the code a few times but it's a bit unclear to me which situation this is handling.

Reply:
It's to show the file's entire path. I made it only do that if the filename is shared with another file.